scripts/setup.sh

```diff
@@ -100,3 +100,47 @@ else
   # Mac OS.
   export CHROME_BIN="/Applications/Google Chrome.app/Contents/MacOS/Google Chrome"
 fi
+
+# Check the version of Python. If its version does not start with 2.7, then
+# try and see whether the 'python2.7' command exists.
+# Borrowed from bhenning's solution for checking python version
+# http://stackoverflow.com/questions/592620
```

Comment:
On second thought, the first `else` condition that catches bad output for `python --version` in this function implicitly tests if python even exists. Unless we want to test for it explicitly (keeping the spirit of PEP 20, even though this is bash), I think we can leave this as is. 

This would break if the user actually ran some output that echoed something like `Python 2.7.6`, but that seems like it would have to be more intentional than accidental.